Sources/Fluent/Query/Join.swift

```diff
@@ -79,17 +80,17 @@ public final class Pivot<
 
     public func makeNode(context: Context = EmptyNode) throws -> Node {
         return try Node(node: [
-            "id": id,
-            "\(type(of: self).left.name)_id": leftId,
-            "\(type(of: self).right.name)_id": rightId,
+            "\(idKey)": id,
+            "\(type(of: self).left.name)\(idKey)": leftId,
```

Comment:
There should be an underscore between these to variables.

```swift
"\(type(of: self).left.name)_\(idKey)"
```

Otherwise it won't line up with what `Union` is expecting.